src/pages/DetailPage.spec.js

```diff
@@ -0,0 +1,25 @@
+import { render, screen } from "@testing-library/react";
+import DetailPage from "./DetailPage";
+import userEvent from "@testing-library/user-event";
+
+describe("DetailPage", () => {
+  it("renders an title, shows the artist and has a button", () => {
```

Comment:
The behavior of the button is missing here, maybe you can add it before the button const  to mark it off as an extra test?